packages/components/src/focal-point-picker/styles/focal-point-style.ts

```diff
@@ -3,45 +3,30 @@
  */
 import styled from '@emotion/styled';
 
-/**
- * WordPress dependencies
- */
-import { Path, SVG } from '@wordpress/primitives';
-
-/**
- * Internal dependencies
- */
-import { COLORS } from '../../utils';
-
 export const FocalPointWrapper = styled.div`
 	background-color: transparent;
 	cursor: grab;
-	height: 30px;
-	margin: -15px 0 0 -15px;
-	opacity: 0.8;
+	height: 48px;
+	margin: -24px 0 0 -24px;
 	position: absolute;
 	user-select: none;
-	width: 30px;
+	width: 48px;
 	will-change: transform;
 	z-index: 10000;
 
 	${ ( { isDragging }: { isDragging: boolean } ) =>
 		isDragging && 'cursor: grabbing;' }
 `;
 
-export const PointerIconSVG = styled( SVG )`
+export const PointerCircle = styled.div`
 	display: block;
 	height: 100%;
 	left: 0;
 	position: absolute;
 	top: 0;
 	width: 100%;
```

Comment:
```suggestion
	height: 100%;
```
Just a little simplification that could be done while we are looking at these styles. I'd even suggest getting rid of the this inner element altogether if there's no interest in making the `clip-path` suggestion of Dan’s work.